wazo_confd/plugins/meeting/resource.py

```diff
@@ -0,0 +1,258 @@
+        super().__init__(service, user_service, hostname, port)
+        self._auth_client = auth_client
+
+    def get_model(self, uuid, user_uuid, **kwargs):
+        return self.service.get_by(uuid=uuid, owner=user_uuid, **kwargs)
+
+    @required_acl('confd.users.me.meetings.{uuid}.read')
+    def get(self, uuid):
+        return super().get(uuid)
+
+    @required_acl('confd.users.me.meetings.{uuid}.update')
+    def put(self, uuid):
+        kwargs = self._add_tenant_uuid()
+        kwargs['user_uuid'] = self._find_user_uuid()
+        model = self.get_model(uuid, **kwargs)
+        self.parse_and_update(model)
+        return '', 204
+
+    @required_acl('confd.users.me.meetings.{uuid}.delete')
+    def delete(self, uuid):
+        kwargs = self._add_tenant_uuid()
+        kwargs['user_uuid'] = self._find_user_uuid()
+        model = self.get_model(uuid, **kwargs)
+        self.service.delete(model)
+        return '', 204
+
+    def _find_user_uuid(self):
+        token = request.headers.get('X-Auth-Token') or request.args.get('token')
+        try:
+            token_infos = self._auth_client.token.get(token)
+        except HTTPError as e:
+            logger.warning('HTTP error from wazo-auth while getting token: %s', e)
+            raise errors.param_not_found('user_uuid', 'meetings')
+
+        user_uuid = token_infos['metadata']['pbx_user_uuid']
+        if not user_uuid:
+            raise errors.param_not_found('user_uuid', 'meetings')
+
+        return user_uuid
+
+
+class UserMeetingList(MeetingList):
+
+    model = Meeting
+
+    def __init__(
+        self,
+        service,
+        user_service,
+        tenant_service,
+        endpoint_sip_service,
+        endpoint_sip_template_service,
+        hostname,
+        port,
+        auth_client,
+    ):
+        super().__init__(
+            service,
+            user_service,
+            tenant_service,
+            endpoint_sip_service,
+            endpoint_sip_template_service,
+            hostname,
+            port,
+        )
+        self._auth_client = auth_client
+
+    def build_headers(self, meeting):
+        return {'Location': url_for('meetings', uuid=meeting.uuid, _external=True)}
+
+    @required_acl('confd.users.me.meetings.create')
+    def post(self):
+        body = self.add_user_to_owner_uuids(request.get_json())
+        return self._post(body)
+
+    @required_acl('confd.users.me.meetings.read')
+    def get(self):
+        return super().get()
+
+    def add_user_to_owner_uuids(self, body):
+        user_uuid = self._find_user_uuid()
+        body['owner_uuids'] = list(set(body.get('owner_uuids', []) + [user_uuid]))
+        return body
+
+    def search_params(self):
+        params = ListSchema().load(request.args)
+        params['owner'] = self._find_user_uuid()
+        return params
+
+
+def random_string(length):
+    return ''.join(random.choice(string.ascii_lowercase) for _ in range(length))
```

Comment:
```suggestion
    return ''.join(secrets.choice(string.ascii_lowercase) for _ in range(length))
```
This is exactly the purpose of the [`secrets` module](https://docs.python.org/3/library/secrets.html)

Reply:
I did not know about the secrets module. thanks